documentation/profiles/index.md

```diff
@@ -141,4 +141,4 @@ STIX profiles are generally created through community agreement (for community p
 
 ## Questions?
 
-Profiles are a complicated topic. Please [get in touch](stix-taxii@hq.dhs.gov) with the team if you have any questions or think this documentation could be improved.
+Profiles are a complicated topic. Please [get in touch](mailto:stix-taxii@hq.dhs.gov) with the team if you have any questions or think this documentation could be improved.
```

Comment:
We probably should make it a configuration option and set it to `stix@mitre.org` for now. I want to make it configuration bc I think we might switch it to the cti-users list at some point.